.github/workflows/p9.yml

```diff
@@ -0,0 +1,38 @@
+name: p9
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+    name: Build
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}    
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
```

Comment:
is there a way to run this before we merge it?

i'm curious about the qemu-based integration test. may want to tag that as linux only.

https://github.com/hugelgupf/p9/blob/main/fsimpl/localfs/localfs_integration_test.go

Reply:
oh yeah we should. let me give it a go :)

Reply:
Yeah the tests are definitely running on all PRs. and failing for both windows and mac. 
I think those should be seperate PRs to fix them tho.

Reply:
https://github.com/sevki/p9/runs/895589927?check_suite_focus=true